obspy/io/sac/sactrace.py

```diff
@@ -1189,14 +1189,14 @@ def _from_arrays(cls, hf=None, hi=None, hs=None, data=None):
 
         .. rubric:: Example
 
-        >>> sac = SACTrace._from_arrays() # doctest: +SKIP
-        >>> print(sac) # doctest: +SKIP
+        >>> sac = SACTrace._from_arrays()
```

Comment:
hehehe `XXX: assumes hf was provided`.  I even told myself this would happen :smile: